code/src/Form.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+
+import { WelcomePage } from './Components/WelcomePage'
+import { TextInput } from './Components/TextInput'
+import { RadioButtons } from './Components/RadioButtons'
+import { SelectDropdownDrink } from './Components/SelectDropdownDrink'
+import { DropdownSnacks } from './Components/DropdownSnacks'
+import { Summary } from './Components/Summary'
+
+// All the useStates
+export const Form = () => { 
+    const [name, setName] = useState('');
+    const [snack, setSnack] = useState ('');
+    const [drink, setDrink] = useState ('');
+    const [game, setGamesList] = useState('');
+    const [section, setSection] = useState('welcomePage');
+
+// Components made into an array to build the multistep 
+
+    const sectionOrder = [
+        'welcomePage',
+        'nameQuestion',
+        'gameQuestion',
+        'drinkQuestion',
+        'snackQuestion',
+        'summary'
+    ];
+
+// Function that prevents refreshing the page, as well as, use of indexOf method to go through the different steps 
+// by the use of useState that starts in the welcome page and then adds +1 moving this way through the array 
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        const nextSectionIndex = sectionOrder.indexOf(section);
+        setSection(sectionOrder[nextSectionIndex+1]);   
+      };
+
+    return (
+    <>
+    <h1 tabIndex='0'>VR Games Night!</h1>
+    <section className='form-container'>
+        {section !== 'summary' ? (
+            <form onSubmit={handleSubmit}>
+                {section === 'welcomePage' && (
+                    <WelcomePage/>   
+                )}
+                {section === 'nameQuestion' && (
+                    <TextInput name= {name} setName= {setName}/>   
```

Comment:
Now I'm being picky, but I think you usually don't make a space after = when writing props, same in the lines below.

Reply:
Not picky at all, it was my mistake! thank you